web/src/emoji_picker.ts

```diff
@@ -287,44 +334,48 @@ function toggle_reaction(emoji_name, event) {
     }
 
     const message_id = current_message_id;
-    const message = message_store.get(message_id);
-    if (!message) {
-        blueslip.error("reactions: Bad message id", {message_id});
-        return;
-    }
+    if (message_id) {
```

Comment:
This should use the early return style, rather than indenting a big block of code here.